lxl-web/src/routes/(app)/[[lang=lang]]/[fnurgel=fnurgel]/+page.svelte

```diff
@@ -12,12 +13,21 @@
 				<DecoratedData data={data.heading} showLabels={ShowLabelsOptions.Never} />
 			</h1>
 		</header>
-		<div class="overview mb-4">
-			<DecoratedData data={data.overview} block />
+		<div class="flex items-start">
+			<div>
+				<div
+					class="rounded-lg h-64 w-64 bg-contain bg-top bg-no-repeat"
+					style="background-image: url('{data.imageUris[0].imageUri}')"
+				/>
```

Comment:
Could we use `<img>` elements instead of using `background-image`? The former are generally better for accessibility and SEO.

A related question: is it easy to also get the width and height of the images? Width and height attributes on img elements are good to have as they prevent layout shifting when the page renders.